### PR TITLE
Answers Week2

### PR DESCRIPTION
**Pull request by vlad-pisotskyi**

### CODE REVIEW

File: 01-node-tutorial/answers/16-streams.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+});
+
+let counter = 0;
+stream.on("data", (result) => {
+	counter += 1;
+	console.log(counter);
+});
+stream.on("end", () => {
+	// 	highWaterMark: 2000, 85 chunks received
+	// 	highWaterMark: 200000, 1 chunks received
```
Comment (alexgs) [inline 01-node-tutorial/answers/16-streams.js]: The `highWaterMark` is the maximum amount of bytes that Node will read in each chunk of the stream.

File: 01-node-tutorial/answers/writeWithPromisesAwait.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+// writeFile returns promises
+
+const writer = async () => {
+	try {
+		await writeFile("./temp.txt", `Hello, I love you, `);
+		await writeFile("./temp.txt", `won't you tell me your name\n`, {
+			flag: "a",
+		});
+		await writeFile("./temporary/temp.txt", `Hello, I love you!`, {
```
Comment (alexgs) [inline 01-node-tutorial/answers/writeWithPromisesAwait.js]: I think this should be `"./temp.txt"` like lines 7-8.

**Review by alexgs — APPROVED** [review]
Looks good!

File: 01-node-tutorial/answers/16-streams.js (hunk 1 above)
Comment (vlad-pisotskyi) [reply 01-node-tutorial/answers/16-streams.js]: Thanks. `Test the program for several values of highWaterMark.` was in the assignment. I didn't know how to indicate it

File: 01-node-tutorial/answers/writeWithPromisesAwait.js (hunk 2 above)
Comment (vlad-pisotskyi) [reply 01-node-tutorial/answers/writeWithPromisesAwait.js]: oh my, you're right. I was confused slightly by where CTD wanted me to save temp.txt, either in ./temporary  folder (where it was created "last week") in /answers for a .gitignore practice. `Then create an async function called writer that takes 0 arguments, and that writes three lines to a file named temp.txt, by calling the writeFile function with await.`. I've forgotten to change all the paths. thanks.

**Pull request closed without merging**